src/Connector.sol

```diff
@@ -26,6 +26,8 @@ contract CentrifugeConnector is Test {
         address token;
         uint256 latestPrice; // [ray]
         uint256 lastPriceUpdate;
+        string tokenName;
```

Comment:
One thought: can you add a todo here like
```suggestion
        // TODO: the token name & symbol need to be stored because of the separation between adding and deploying tranches.
        // This leads to duplicate storage (also in the ERC20 contract), ideally we should refactor this somehow
        string tokenName;
```